packages/expect/src/__tests__/matchers.test.js

```diff
@@ -2240,3 +2321,66 @@ describe('toMatchObject()', () => {
     ).toThrowErrorMatchingSnapshot();
   });
 });
+
+describe('.toHaveBeenCalledWith', () => {
+  [
+    [
+      [1, 2, 3],
+      [1, 2, 3],
+    ],
```

Comment:
This one is similar to https://github.com/facebook/jest/blob/506a0629147b2cad1469709b0b3e40bcaddc1a8d/packages/expect/src/__tests__/spyMatchers.test.ts#L229